activestorage/CHANGELOG.md

```diff
@@ -1,3 +1,7 @@
+*   Improve performance by parallelizing `ActiveStorage::Service::MirrorService#delete` and `ActiveStorage::Service::MirrorService#delete_prefixed`.
+
+    *heka1024*
```

Comment:
We don't really need to add Changelogs for refactorings
ref: https://guides.rubyonrails.org/contributing_to_ruby_on_rails.html#updating-the-changelog

Reply:
Thank you for notice!

Reply:
I just removed changelog for this PR.